src/api/public/apidocs-new/paths/source_project_name_attribute.yaml

```diff
@@ -0,0 +1,54 @@
+get:
+  summary: Get all the project's attributes
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - name: meta
+      in: query
+      schema:
+        type: string
+    - name: rev
+      in: query
+      schema:
+        type: string
+    - name: view
+      in: query
+      schema:
+        type: string
+      description: The value of this parameter should only be `blame`
```

Comment:
When `view=blame` is passed, is the response of the endpoint using the same `source/attributes.yaml` schema?

If not, it could help for completeness to add an example in the `responses:` section.

Reply:
The result is not pure XML but the content type we receive is still `Content-Type: application/xml; charset=utf-8`, so I add an example so the user knows the structure thanks to the value of the example.